main/networking/Runtime.ts

```diff
@@ -254,12 +267,20 @@ class RuntimeConnection {
     return cb(event, ...args);
   };
 
+  sendMessage = (message: Buffer | null, cb?: ((err?: Error | undefined) => void) | undefined) => {
```

Comment:
Wrapper around `this.socket.write` to check for `null` messages which we shouldn't send